manifest.json

```diff
@@ -18,7 +18,7 @@
                 "content.js"
             ],
             "matches": [
-                "https://leetcode.com/problems/*/description/*"
+                "https://leetcode.com/problems/*"

```

Comment:
changed because the count wasn't showing up when a question was opened without `/description/`.